modules/jaeger-deploy-streaming.adoc

```diff
@@ -138,3 +138,4 @@ jaeger-streaming-kafka-0                                          2/2     Runnin
 jaeger-streaming-query-65bf5bb854-ncnc7                           3/3     Running   0          80s
 jaeger-streaming-zookeeper-0                                      2/2     Running   0          3m39s
 ----
+
```

Comment:
You removed the note here, which is the correct topic for the streaming deployment, but probably the wrong place.  If you look at the preview you'll get a better sense of the flow of this assembly.

Rather than putting the note at the end of the procedure (where users won't see it until AFTER they've already tried to deploy) I would put it before the procedure, right after the prerequisites but before the procedure.

https://deploy-preview-28313--osdocs.netlify.app/openshift-enterprise/latest/jaeger/jaeger_install/rhbjaeger-deploying.html#jaeger-deploy-streaming_jaeger-deploying